SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -384,7 +385,11 @@ Quarks {
 		});
 	}
 	*isPath { |string|
-		^string.findRegexp(regex.isPath).size != 0
+		if(thisProcess.platform.name !== 'windows', {
+			^string.findRegexp(regex.isPath).size != 0
+		}, {
+			^(string.findRegexp(regex.isPath).size != 0).and(string.findRegexp(regex.isURL).size == 0)
+		});
```

Comment:
but how can a URL ever match "\\\\|/" ? are you sure that's what happened with that example you sent me ?